src/keboola/utils/formatter.py

```diff
@@ -0,0 +1,15 @@
+import string
+from typing import List
+
+PERMITTED_CHARS = string.digits + string.ascii_letters + '_'
+
+
+def normalize_header(header: List[str], permitted_chars: str = PERMITTED_CHARS) -> List[str]:
```

Comment:
@bakobako prazdny string je taky neplatny header takze `"#%^"` spadne pri importu.

Napada me, jestli to neudelat chytrejsi. Tj. mit nekolik strategii jak tu header normalizovat -> mohla by to byt zas Factory. Varianty:
1. Nahradit nahradit neplatny znaky empty stringem 
2. Nahradit neplatny znaky `_`
3. Nahradit mezery `_` a neplatny znaky zahodit (mohl by byt default)
4. Nahradit neplatny znaky unique nazvem - treba pomoci unicode id `ord('#')` a oddelit `_`
5. Custom mapping pomoci dictionary `{'#':'_hsh_'}`

Potom to cely muzeme vzit a udelat z toho processor. Ten by mohl byt uzitecnej. 

Co myslis?